python/ray/_raylet.pyx

```diff
@@ -1907,6 +1907,26 @@ cdef class CoreWorker:
 
         return ref_counts
 
+    def get_inflight_tasks_count(self):
+        cdef:
+            unordered_map[c_string, c_vector[size_t]] c_tasks_count
+
+        c_tasks_count = (
+            CCoreWorkerProcess.GetCoreWorker().GetInflightTasksCount())
+        it = c_tasks_count.begin()
+
+        tasks_count = dict()
+        while it != c_tasks_count.end():
+            func_name = <unicode>dereference(it).first
+            counters = dereference(it).second
+            tasks_count[func_name] = {
+                "received": counters[0],
+                "executing": counters[1],
+                "executed": counters[2],
```

Comment:
Can we rename these to QUEUED, RUNNING, FINISHED? Executing/executed is easy to be confused since they are close to each other in characters.